### ISSUE TITLE
Set.seed option

### ISSUE THREAD

**Issue by wmotte** [NLeSC/mcfly#277]
Dear developers,
Thanks for this very nice and well-documented software.
I use it for student teaching.
In some cases I would like students to compare model output.
However, due to the random hyperparameter selection every run results in different outputs.
My questions is, would it be possible to add a set.seed option to mcfly?
I have tried to set the tensorflow random number seed, but that does not affect the hyperparameter selection.
Thanks!

**Comment by cwmeijer** [comment]
Thanks for submitting an issue and your interest in mcfly. The hyperparameters are selected using the random number generator in numpy. ```numpy.random.seed(1234)``` should do the trick. See for example the [mcfly tutorial](https://github.com/NLeSC/mcfly-tutorial/blob/master/notebooks/tutorial/tutorial.ipynb).
